tools/serverpod_cli/lib/src/language_server/language_server.dart

```diff
@@ -112,12 +122,22 @@ Future<void> runLanguageServer() async {
   await connection.listen();
 }
 
-void _sendServerDisabledNotification(Connection connection) {
+void _sendModulesNotFoundNotification(Connection connection) {
   connection.sendNotification(
     'window/showMessage',
     ShowMessageParams(
       message:
-          'Serverpod protocol validation disabled, not a Serverpod project.',
+          'Serverpod model validation disabled. Unable to locate necessary modules, have you run dart pub get?',
```

Comment:
Suggestion: Add a symbol like `"` around the command to differentiate it from the rest of the text.